ansible/README.md

```diff
@@ -148,6 +148,58 @@ ansible-playbook -i environments/$ENVIRONMENT prereq.yml
 
 **Hint:** During playbook execution the `TASK [prereq : check for pip]` can show as failed. This is normal if no pip is installed. The playbook will then move on and install pip on the target machines.
 
+### [Optional] Enable the new scheduler
+
+You can enable the new scheduler of OpenWhisk.
+It will run one more component called "scheduler" and ETCD.
+
+#### Configure service providers for the scheduler
+You can update service providers for the scheduler as follows. 
+
+**common/scala/src/main/resources**
+```
+whisk.spi {
```

Comment:
shouldn't a provider for the scheduler db required here? i.e. `ETCDProvider`?

Reply:
Do we have the `ETCDProvider`?
If we need to use a key-value store other than ETCD, we can introduce the layer too.
But for now, I believe the code is sticking with ETCD.